pkg/util/filehelper/file.go

```diff
@@ -152,3 +152,24 @@ func CopyDir(src string, dst string) (err error) {
 
 	return
 }
+
+// IsDirEmpty checks whether directory is empty or not
+func IsEmptyDir(path string) bool {
+	if !Exists(path) {
+		return true
```

Comment:
@hferentschik , if path doesn't exists then we assume file is empty and hence true.

If we return _false_, the meaning would be "If there doesn't exist path for directory, return _IsEmptyDir_ as _false_ which means "Directory contains something".